ios/MusicControlManager.m

```diff
@@ -85,23 +85,28 @@ - (dispatch_queue_t)methodQueue
 
         [details setValue:speed forKey:MEDIA_SPEED];
     }
-    if ([state isEqual:MEDIA_STATE_STOPPED]) {
-        MPRemoteCommandCenter *remoteCenter = [MPRemoteCommandCenter sharedCommandCenter];
-        [self toggleHandler:remoteCenter.stopCommand withSelector:@selector(onStop:) enabled:false];
-    }
 
+    if ([state isEqual:MEDIA_STATE_STOPPED]) {
```

Comment:
It doesn't actually look like there's any changes here, can you please drop the change so it doesn't show up in the diff?